neqo-transport/src/server.rs

```diff
@@ -174,8 +167,8 @@ pub struct Server {
     active: HashSet<ActiveConnectionRef>,
     /// The set of connections that need immediate processing.
     waiting: VecDeque<StateRef>,
-    /// Outstanding timers for connections.
-    timers: Timer<StateRef>,
+    /// The latest [`Output::Callback`] returned from [`Server::process`].
+    callback: Option<Instant>,
```

Comment:
Maybe "wake_at" or something like that.  "callback" implies that this is a function.

Reply:
:+1: done via https://github.com/mozilla/neqo/pull/1784/commits/32db5153ef784f46d8745a2fd91b7aa94f2f7b8c.